### DISCUSSION TITLE
Option to change colour on edit button

### DISCUSSION THREAD

**Discussion by kilim1983** [YetiForceCompany/YetiForceCRM#17471] [How to]
Hello,

We us CRM ver. 6.5. I want to know if  is there a simple option to change the color of the Edit button in different modules (contacts, Leade, Tasks etc.). This button is used very often. Button like Archve (Blue), Trash (Red) Delete (Black)-the buttons are colorful.

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/44589374/c0ff4618-ce73-45e3-abd8-f54470b30a30)

Regards,
Damian

**Comment by rskrzypczak** [comment]
You can change the color in the configuration file. 
`config/Search.php`
https://doc.yetiforce.com/code/classes/Config-Search.html#property_LIST_ENTITY_STATE_COLOR
However, these are global settings.

**Reply by kilim1983** [reply]
Hello,

I was try to change this by edit this:

public static mixed $LIST_ENTITY_STATE_COLOR = ['Archived' => '#0032a2', 'Trash' => '#ab0505', 'Active' => '#009405']

I added/changed:

_public static mixed $LIST_ENTITY_STATE_COLOR = ['Archived' => '#0032a2', 'Trash' => '#ab0505', 'Active' => '#009405', 'Edit' => '#8BC34A']_ 

or I added/changed

_public static mixed $LIST_ENTITY_STATE_COLOR = ['Archived' => '#0032a2', 'Trash' => '#ab0505', 'Active' => '#009405', 'Edit field' => '#8BC34A']_ 

but this is not working on my case.

Where am I making a mistake.

Regards,
Damian

**Comment by rskrzypczak** [comment]
I answered too quickly. There is no way to manage button colors in the system. The mentioned configuration includes these three buttons from the record state, which is an exception.

All you have to do is modify the global *.css file, which I do not recommend, because it will be overwritten with the next update, or create your own css file and upload it to the system.
